t/plugin/jwt-auth.t

```diff
@@ -1140,3 +1140,73 @@ base64_secret required but the secret is not in base64 format
 [error]
 --- request
 GET /t
+
+
+
+=== TEST 50: when the exp value is not set, make sure the default value(86400) works
```

Comment:
@membphis do these two cases cover the scenarios you mention?
I'm not sure if the TEST51 is elegant enough.